core/docs/spark-qualification-tool.md

```diff
@@ -32,50 +20,70 @@ applicable environments.  Here are the cluster information for the ETL benchmark
 
-## Running the Qualification tool standalone for CSP environments on Spark event logs
-### User Tools Prerequisites and Setup for CSP environments
+## Running the Standalone Qualification Tool for CSP Environments on Spark Event Logs
+### User Tools Prerequisites and Setup for CSP Environments
 
-* [Dataproc](https://github.com/NVIDIA/spark-rapids-tools/blob/main/user_tools/docs/user-tools-dataproc.md)
-* [EMR](https://github.com/NVIDIA/spark-rapids-tools/blob/main/user_tools/docs/user-tools-aws-emr.md)
-* [Databricks AWS](https://github.com/NVIDIA/spark-rapids-tools/blob/main/user_tools/docs/user-tools-databricks-aws.md)
+* [Dataproc](../../user_tools/docs/user-tools-dataproc.md)
+* [EMR](../../user_tools/docs/user-tools-aws-emr.md)
+* [Databricks AWS](../../user_tools/docs/user-tools-databricks-aws.md)
 
 ### Qualify CPU Workloads for Potential Cost Savings and Acceleration with GPUs
 
-The qualification tool will run against logs from your CSP environment and then will output the applications
-recommended for acceleration along with estimated speed-up and cost saving metrics.
+The currently supported CSPs are *dataproc*, *emr*, and *databricks-aws*. The Qualification tool will run against logs from the specific CSP environment and will output GPU acceleration recomendations with estimated speed-up and cost saving metrics.
 
-Usage: `spark_rapids_user_tools <CSP> qualification --cpu_cluster <CLUSTER> --eventlogs <EVENTLOGS-PATH>`
+Usage:
+```bash
+# CSP provider ID
+export CSP=<dataproc | emr | databricks-aws>
+# The live cluster's name or the cluster configuration file detailing cluster instances and cluster size
+export CLUSTER=<CSP hosted cluster>
+# Storage location for Spark event logs
+# For Dataproc, a GCS path
+# For EMR or Databricks-AWS, a S3 path
+export EVENTLOGS_PATH=<path to application event logs>
 
-The supported CSPs are *dataproc*, *emr*, and *databricks-aws*.  The EVENTLOGS-PATH should be the storage location
-for your eventlogs.  For Dataproc, it should be set to the GCS path.  For EMR and Databricks-AWS, it should be set to
-the S3 path.  THE CLUSTER can be a live cluster or a configuration file representing the cluster instances and size. 
-More details are in the above documentation links per CSP environment
+spark_rapids_user_tools ${CSP} qualification \
+   --cpu_cluster ${CLUSTER} \
+   --eventlogs ${EVENTLOGS_PATH}
+```
 
-Help (to see all options available): `spark_rapids_user_tools <CSP> qualification --help`
+> NOTE: If using S3, make sure the additional [S3 configuration steps](#S3-file-path-support-and-configuration) have been completed. More CSP specific configuration details are covered in the [linked documentation](#User-Tools-Prerequisites-and-Setup-for-CSP-Environments) for CSP specific configuration.
```

Comment:
same issue here. We are overlapping with content of user-tools documentation.
And it sounds like biased/favoring S3. if we mention s3, then we probably need to list all the cloud storages